.gitignore

```diff
@@ -21,3 +24,6 @@
 
 # Ignore Byebug History
 .byebug_history
+
+# Ignore Emacs stuff
+*~
```

Comment:
I'm not opposed to this but wondering if it's something that can be put in your global git ignore instead?